### PR TITLE
mecab: merge mecab-unidic as an optional resource

### PR DESCRIPTION
**Pull request by vszakats**
It's my first attempt to tackle a resource and it even has an optional twist and a caveat. Ergo: This can be completely wrong. Please be patient.

Other updates:
- mecab-unidic: update url and checksum
- mecab: add desc to pass strict audit

### CODE REVIEW

File: Library/Formula/mecab.rb [hunk 1]
```diff
@@ -12,7 +13,31 @@ class Mecab < Formula
...
+                                "--prefix=#{prefix}"
+          system "make", "install"
+        end
+      end
+
+      def caveats; <<-EOS.undent
+        If you want to use UniDic, please rewrite "dicdir".
+           #{opt_prefix}/etc/mecabrc
+        EOS
+      end
```
Comment (tdsmith) [inline Library/Formula/mecab.rb]: Nesting a caveats method under the install method shouldn't do anything.

File: Library/Formula/mecab.rb [hunk 2]
```diff
@@ -12,7 +13,31 @@ class Mecab < Formula
     sha1 "9747369cd4c0aa246e6a973c4f2e5652e174bae8" => :mountain_lion
   end
 
+  option "with-unidic", "UniDicとは"
```
Comment (tdsmith) [inline Library/Formula/mecab.rb]: 英語で書いたらほうがいいと思う、おねがいします

File: Library/Formula/mecab.rb (hunk 1 above)
Comment (tdsmith) [inline Library/Formula/mecab.rb]: Can the formula take care of the configuration that's required?

**Comment by tdsmith** [comment]
Adding dictionaries to the mecab formula makes sense to me.

File: Library/Formula/mecab.rb (hunk 1 above)
Comment (vszakats) [inline Library/Formula/mecab.rb]: This dictionary has the required logic:
https://github.com/Homebrew/homebrew/blob/master/Library/Formula/mecab-ko-dic.rb#L18-L20

But, what to do when multiple dictionaries are selected?

**Comment by vszakats** [comment]
Abandoning this along with https://github.com/Homebrew/homebrew/pull/40238.

**Pull request closed without merging**